From 25444e136d3a545775a8efe0e3a3b75ac472f034 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Mon, 12 Aug 2024 15:38:12 +0530
Subject: [PATCH] camera: use default vendor hal

As Google's default HAL is used, changed the package to Google's
default camera hal.

Tests done:
- Android boot
- Camera preview and capture

Tracked-On: OAM-123336
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 groups/camera-ext/ext-camera-only/product.mk | 2 +-
 groups/device-specific/caas/product.mk       | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/groups/camera-ext/ext-camera-only/product.mk b/groups/camera-ext/ext-camera-only/product.mk
index b5d3f00..585908a 100644
--- a/groups/camera-ext/ext-camera-only/product.mk
+++ b/groups/camera-ext/ext-camera-only/product.mk
@@ -6,7 +6,7 @@ PRODUCT_COPY_FILES += \
 
 
 # External camera service
-PRODUCT_PACKAGES += android.vendor.hardware.camera.provider-V1-external-service
+PRODUCT_PACKAGES += android.hardware.camera.provider-V1-external-service
 
 #VHAL camera
 PRODUCT_PACKAGES += camera.$(TARGET_BOARD_PLATFORM) \
diff --git a/groups/device-specific/caas/product.mk b/groups/device-specific/caas/product.mk
index dc4b6ff..2c1dfe9 100755
--- a/groups/device-specific/caas/product.mk
+++ b/groups/device-specific/caas/product.mk
@@ -24,8 +24,8 @@ PRODUCT_PACKAGES += ClipboardAgent
 
 PRODUCT_PACKAGES +=  \
                     android.hardware.usb-service.example \
-                    camera.vendor.device@1.0-impl \
-                    android.hardware.camera.provider@2.4-impl-intel \
+                    camera.device@1.0-impl \
+                    android.hardware.camera.provider@2.4-impl \
                     android.hardware.graphics.allocator-service.minigbm \
                     android.hardware.renderscript@1.0-impl \
                     android.hardware.identity \
-- 
2.45.2

